src/account/SignUp.jsx

```diff
@@ -44,30 +43,34 @@ export default function SignUpScreen(props) {
 
     <div>
       <h1>Sign Up Screen!</h1>
```

Comment:
You did not navigate the user to the (new) Sign Up page. The user is still on the account page. When a user clicks the Sign Up button, you should navigate the user to the Sign Up page.

Reply:
I'm not sure what you mean. When you click on the button, it renders the Sign Up page. Similar logic as the Sign In page.

Reply:
For example, a user clicks the Sign Up button and should be navigated to a new page (Sign Up page). The user can enter his/her email and password to sign up on this new page.
It will be better to create a new page to handle user sign up / sign in. Because in the future, we may add "reset password," "forgot password," "enter verification code," ... features/components on the Sign Up/Sign In page.
Also, we will add more features / components on the Account page. So, please do not render the sign up / sign in form on the Account page. Create new pages instead.